packages/webpack-extensive-lodash-replacement-plugin/index.js

```diff
@@ -443,15 +478,22 @@ class ExtensiveLodashReplacementPlugin {
 	}
 
 	apply( compiler ) {
+		compiler.hooks.thisCompilation.tap( 'LodashReplacementPlugin', compilation => {
+			this.compilation = compilation;
+		} );
+
 		compiler.hooks.normalModuleFactory.tap( 'LodashReplacementPlugin', nmf => {
-			nmf.hooks.beforeResolve.tap( 'LodashReplacementPlugin', result => {
+			this.moduleResolver = this.moduleResolver || nmf.getResolver( 'normal' );
+			this.initBaseLodashData();
```

Comment:
This initialization is async and should be awaited somewhere to prevent race conditions.

Reply:
Argh, I shouldn't have missed that. Thanks, fixed!